src/components/settings/SettingsSSOTab/steps/NewSSOConfigConfigureStep.tsx

```diff
@@ -15,39 +21,39 @@ const SSOConfigConfigureStep = () => {
       </p>
       <Form.Group className="mb-4">
         <ValidatedFormControl
-          formId="DummyOAuthUserID"
+          formId="user_id_attribute"
           type="text"
           floatingLabel="User ID"
           fieldInstructions="URN of the SAML attribute that edX can use as a unique, persistent user ID."
         />
       </Form.Group>
       <Form.Group className="mb-4">
         <ValidatedFormControl
-          formId="DummyOAuthFullName"
+          formId="full_name_attribute"
           type="text"
           floatingLabel="Full Name"
           fieldInstructions="URN of SAML attribute containing the user's full name."
         />
       </Form.Group>
       <Form.Group className="mb-4">
         <ValidatedFormControl
```

Comment:
Is the username the first name? Or should there be a separate username form field like there is in Figma?

Reply:
we should follow what the [orchestrator docs lay out](https://gist.github.com/praveenram2u/a31b3e33b7ef4f5e550a8f196bb61aed#file-sso-self_service_orchestration_api-yaml-L83) (and also what our backend has in the tables). So full, first and username are all valid